setup.md

```diff
@@ -117,6 +111,14 @@ $ flutter doctor
 The first time your run the `flutter` command, it will download its dependencies and compile
 itself. Subsequent runs should be much faster.
 
+If `flutter doctor` prints all checkmarks (`[✓]`), then you are cleared
+for takeoff!
+
+If `flutter doctor` displays one or more errors (`[x]`), then one or more
```

Comment:
Note: it doesn't display `[x]`'s - just

- `[✓]` (A-OK)
- `[-]` (partially installed)
- `[ ]` (not installed)